### DISCUSSION TITLE
How to set the default theme as dark?

### DISCUSSION THREAD

**Discussion by ticoeteco23gb** [yokoffing/Betterfox#298] [Q&A] [answered]
Hello, I started using BF recently, but it seems like when I open and close the browser, the light theme is the default one.
It hurts my eyes, since I use my computer on the nighttime, and I shouldn't lower too much the brightness of the screen.

**Comment by yokoffing** [comment, marked as answer]
https://github.com/yokoffing/Betterfox/issues/285#issuecomment-2038416307

**Reply by ticoeteco23gb** [reply]
Thank you so much!! That's really helpful.
